packages/core/src/messages/packet.ts

```diff
@@ -48,6 +49,8 @@ export function validateCreatedTicket(myBalance: BN, ticket: Ticket) {
   }
 }
 
+const TICKET_BASE_UNIT = new BN(new BigNumber(TICKET_AMOUNT).div(TICKET_WIN_PROB).toString())
```

Comment:
Why not just use BN here?

Reply:
When using probabilities != 1, such as 0.4, we need at least one division - in this case by 10 or by 5 - and in order to determine the value that we write into the ticket, we need to include the roundings that are happening due to the division.

BN.js does not support floats and its division logic leads to unwanted results, which made me use BigNumber.js instead of implementing a fancy division with integers to circumvent floats

Reply:
obsolete now, using inverseProbs